Poker.BE/Poker.BE.Domain/Game/Hand.cs

```diff
@@ -1,106 +1,214 @@
+        {
+            PlaceAntes();
+            PlaceSmallBlind();
+            PlaceBigBlind();
+        }
+
+        /// <summary>
+        /// If needed:
+        /// All of the active players are forced to pay some blind
+        /// payment to the pot, regardless to the regular blinds.
+        /// </summary>
+        private void PlaceAntes()
+        {
+            if (gameConfig.AntesValue == 0)
+                return;
+
+            //first player has to "bet" the ante
+            if (CurrentRound.CurrentPlayer.Wallet.AmountOfMoney < gameConfig.AntesValue)
+                CurrentRound.PlayMove(Round.Move.Allin, gameConfig.AntesValue);
+            else
+                CurrentRound.PlayMove(Round.Move.Bet, gameConfig.AntesValue);
+
+            //other players "call" the ante
+            for (int i = 1; i < CurrentRound.ActiveUnfoldedPlayers.Count; i++)
+            {
+                if (CurrentRound.CurrentPlayer.Wallet.AmountOfMoney < gameConfig.AntesValue)
+                    CurrentRound.PlayMove(Round.Move.Allin, gameConfig.AntesValue);
+                else
+                    CurrentRound.PlayMove(Round.Move.Call, gameConfig.AntesValue);
+            }
+        }
+
+        private void PlaceSmallBlind()
+        {
+            if (CurrentRound.CurrentPlayer.Wallet.AmountOfMoney < gameConfig.MinimumBet / 2)
+                CurrentRound.PlayMove(Round.Move.Allin, gameConfig.MinimumBet / 2);
+            else
+                CurrentRound.PlayMove(Round.Move.Raise, gameConfig.MinimumBet / 2);
+        }
+
+        private void PlaceBigBlind()
+        {
+            if (CurrentRound.CurrentPlayer.Wallet.AmountOfMoney < gameConfig.MinimumBet)
+                CurrentRound.PlayMove(Round.Move.Allin, gameConfig.MinimumBet);
+            else
+                CurrentRound.PlayMove(Round.Move.Raise, gameConfig.MinimumBet);
+        }
+
+        private void Showdown()
+        {
+            throw new NotImplementedException();
+        }
+
+        private void PickAWinner()
+        {
+            Pot lastPot = CurrentRound.CurrentPot;
+            List<Player> potWinners;
+            //pick winner for each pot separately
+            while (lastPot.PartialPot != null)
+            {
+                PickAWinner pickPotWinner = new PickAWinner(lastPot.PlayersClaimPot, communityCards);
+                potWinners = pickPotWinner.GetWinners();
+
+                //divide pot money to winning players
+                foreach (Player player in potWinners)
+                {
+                    if (!WinnersProfits.ContainsKey(player))
+                    {
+                        WinnersProfits.Add(player, 0);
+                    }
+                    WinnersProfits[player] += lastPot.Value / potWinners.Count;
+                    player.AddMoney(WinnersProfits[player]);
+                }
+
+                lastPot = lastPot.PartialPot;
+            }
+        }
+        #endregion
+    }
```

Comment:
❗️ big change here and its hard to see what's had changed with `git diff` - can you explain briefly what had changed?